src/drag-and-drop/components/container.tsx

```diff
@@ -63,7 +86,20 @@ export const Container: React.FC<IProps> = ({ authoredState, interactiveState, s
         };
       }
     });
-  }, [authoredState.draggableItems]);
+    authoredState.dropZones?.forEach(target => {
+      if (target.imageUrl) {
+        const img = document.createElement("img");
+        img.src = target.imageUrl;
+        img.onload = () => {
+          setTargetDimensions(prevHash => ({...prevHash, [target.id]: {width: target.targetWidth || img.width, height: target.targetHeight || img.height }}));
```

Comment:
based on spacing you probably want:
`setTargetDimensions(prevHash => ({...prevHash, [target.id]: {width: target.targetWidth || img.width, height: target.targetHeight || img.height}}));`

Reply:
In other words, make sure spacing around brackets is the same.
this:
`{ foo: pizza }`
or this:
`{foo: pizza}`

But not these:
`{foo: pizza }`
`{ foo: pizza}`

Reply:
And a note here, you can either have the space at the beginning/end or not have the space.  Doesn't matter as long as it is consistent.  I would follow whatever is most common in the project.